charts/reaper-operator/crds/reaper.yaml

```diff
@@ -15,99 +13,156 @@ spec:
+                          namespace:
+                            description: If empty we could default the Reaper namespace.
+                            type: string
+                        required:
                         - name
-                      type: object
-                    cassandraUserSecretName:
-                      type: string
-                    keyspace:
-                      description: Defaults to reaper
-                      type: string
-                    replication:
-                      description: Specifies the replication strategy for a keyspace
-                      properties:
-                        networkTopologyStrategy:
-                          additionalProperties:
+                        type: object
+                      cassandraUserSecretName:
+                        type: string
+                      keyspace:
+                        description: Defaults to reaper
+                        type: string
+                      replication:
+                        description: Specifies the replication strategy for a keyspace
+                        properties:
+                          networkTopologyStrategy:
+                            additionalProperties:
+                              format: int32
+                              type: integer
+                            description: Specifies the replication_factor when NetworkTopologyStrategy
+                              is used. The mapping is data center name to RF.
+                            type: object
+                          simpleStrategy:
+                            description: Specifies the replication_factor when SimpleStrategy
+                              is used
                             format: int32
                             type: integer
-                          description: Specifies the replication_factor when NetworkTopologyStrategy
-                            is used. The mapping is data center name to RF.
-                          type: object
-                        simpleStrategy:
-                          description: Specifies the replication_factor when SimpleStrategy
-                            is used
-                          format: int32
-                          type: integer
-                      type: object
-                  required:
+                        type: object
+                    required:
                     - cassandraDatacenter
                     - replication
+                    type: object
+                  jmxUserSecretName:
+                    description: Defines the username and password that Reaper will
+                      use to authenticate JMX connections to Cassandra clusters. These
+                      credentials need to be stored on each Cassandra node.
+                    type: string
+                  storageType:
+                    type: string
+                type: object
+              tolerations:
```

Comment:
Cool stuff but surprised to see it here since the commits and PR only mention the addition of autoscheduling properties.